core/spec.md

```diff
@@ -2353,6 +2361,25 @@ Regardless of how the model is retrieved, the overall format is as follows:
 ```
 
 The following describes the attributes of Registry model:
+
+- `modelversion` <span id="model.modelversion"></span>
+  - Type: String.
+  - OPTIONAL.
```

Comment:
let's remove the periods on these 2 and on the next attribute - just for consistency with the other attributes

Reply:
Will do. I added them because the previous location had the periods consistently, then I moved it. 

I noticed also some other differences between those two places, but I don't think anyone would notice if they don't compare 1:1

Reply:
crap - add them back in - I just looked and you're right. I'll add then in other spots in the spec to be consistent

Reply:
done